changelogs/fragments/52-netscaler_nitro_request-use-all-filters.yaml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - netscaler_nitro_request - Use all filters for get_filtered instead of only the first one (https://github.com/ansible-collections/community.network/issues/48)
```

```suggestion
  - netscaler_nitro_request - use all filters for get_filtered instead of only the first one (https://github.com/ansible-collections/community.network/issues/48).
```
(I should have linked to the [latest dev version of the fragment guide](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to)...)